src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -106,15 +126,30 @@ const InstancePage = (): JSX.Element => {
   }, [processorListResponse]);
 
   useEffect(() => {
-    if (bridgeError) {
-      console.error(bridgeError);
-      goToHome();
+    if (bridgeError && axios.isAxiosError(bridgeError)) {
+      if (
+        isServiceApiError(bridgeError) &&
+        getErrorCode(bridgeError) === APIErrorCodes.ERROR_4
+      ) {
+        history.replace("/instance-not-found", {
+          title: t("instance.notFound"),
+          message: t("instance.errors.cantFindInstance"),
+        });
+      } else {
+        throw new ErrorWithDetail(
+          getPageTitle(),
+          t("instance.errors.instanceDetailsGenericError")
+        );
+      }
     }
 
     if (processorsError) {
-      console.error(processorsError);
+      throw new ErrorWithDetail(
+        getPageTitle(bridge),
+        t("instance.errors.processorsListGenericError")
+      );
```

Comment:
During development, I found that replacing just the table with a message would lead to an overcomplicated design and implementation, with essentially no benefits: What are we missing? Just a breadcrumb and tabs? To accomplish a case that should not happen, I thought it was worth keeping the page title with just a message.